src/tests/end-to-end/test-resources/image/image-text.html

```diff
@@ -66,9 +66,9 @@ <h1>Non-Decorative text</h1>
         <h1>hidden element</h1>
         <ul>
             <li>
-                <img id="hidden-1" src="img1.png" style="visibility:hidden" />
-                <img id="hidden-2" src="img1.png" style="display:none" />
-                <image id="hidden-3" src="img1.png" style="display:none" />
+                <img id="hidden-1" src="img1.png" style="visibility: hidden;" />
+                <img id="hidden-2" src="img1.png" style="display: none;" />
+                <image id="hidden-3" src="img1.png" style="display: none;" />
```

Comment:
Is this right?? It seems like prettier is changing strings content.

Reply:
I thought so too at first, but it's actually changing a literal `.html` file we have in our end to end test resources; it looks like it's left the content template strings alone